### DISCUSSION TITLE
A real exploit unlike the "new exploit found" discussion that enables ChromeOS developer mode (repost for correct category)

### DISCUSSION THREAD

**Discussion by HelloIloveGimkit2** [3kh0/ext-remover#1295] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
[admvp (Activating Developer Mode Via Powerwash)](https://admvp.vercel.app/)
I did steal the sh1mmer website source code but idrc

**Comment by S-PScripts** [comment]
there is definitely an exploit already about this

**Comment by akabutnicer** [comment]
unfortunately, this was already discovered

**Comment by Charlieis1nsan3** [comment]
see how it feels getting cooked  dumb ah n****.

**Comment by awsome-webdev** [comment]
Charlie gettin with it

**Comment by py660** [comment]
If you were already informed that this exploit was already found and is a duplicate, you probably shouldn't be reposting it, even for organizational purposes.

**Reply by HelloIloveGimkit2** [reply]
it doesn't really matter

**Reply by awsome-webdev** [reply]
He just explained why it does, are you autistic?

**Comment by Charlieis1nsan3** [comment]
fucking stupid bitch it was already found dumb bitch how you like that bitch!

**Reply by akabutnicer** [reply]
real

**Reply by HelloIloveGimkit2** [reply]
> fucking stupid bitch it was already found dumb bitch how you like that bitch!

uh oh he's angry he's gonna go super saiyan

**Reply by akabutnicer** [reply]
> > fucking stupid bitch it was already found dumb bitch how you like that bitch!
> 
> uh oh he's angry he's gonna go super saiyan

he's...mocking you.

**Reply by Charlieis1nsan3** [reply]
> > fucking stupid bitch it was already found dumb bitch how you like that bitch!
> 
> uh oh he's angry he's gonna go super saiyan

i'd like to see you say that shit IRL n****.

**Reply by HelloIloveGimkit2** [reply]
mad already

**Reply by akabutnicer** [reply]
ironic but ok

**Reply by Charlieis1nsan3** [reply]
> mad already

nope

**Reply by py660** [reply]
is it ethical to report users on github or nah

**Reply by akabutnicer** [reply]
> is it ethical to report users on github or nah

just leave the kids alone; it's not worth it.

**Reply by Luigi-Master** [reply]
> > fucking stupid bitch it was already found dumb bitch how you like that bitch!
> 
> uh oh he's angry he's gonna go super saiyan

LOL, THIS LITERALLY MADE ME LAUGH FOR TWO MINUTES! 🤣
